### ISSUE TITLE
Please add separate dev and prod configs

### ISSUE THREAD

**Issue by frankiesardo** [bhauman/figwheel-main-template#19]
Hi @bhauman 

Thanks for providing this template, it's really useful to get up to speed with `fighweel-main`. However we find it really hard to extend since the `min` task basically takes the `dev` configuration and turns the `advanced` optimisation on.

Once you start to add basic things like `closure-defines {goog.DEBUG false}` you really need two separate configs, and once you have two configs you start to encounter other problems like differently named .js files and once you fix that via `output-dir` you break other things like `auto-testing`.

To summarise, it would be nice to have a full production-ready template that is easy to extend 🙏 

Thank you!
